src/scancode/cli.py

```diff
@@ -44,6 +44,8 @@
 import click
 from click.termui import style
 
+from math import floor
```

Comment:
This is a stdlib import and should go with other stdlib imports, before `import os`